x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -500,6 +513,93 @@ export class AlertsClient {
     }
   }
 
+  public async getAlertSummary({
+    gte,
+    lte,
+    featureIds,
+    filter,
+    fixedInterval = '1m',
+  }: GetAlertSummaryParams) {
+    try {
+      const indexToUse = ((await this.getAuthorizedAlertsIndices(featureIds)) ?? []).join(',');
+
+      // first search for the alert by id, then use the alert info to check if user has access to it
+      const responseAlertSum = await this.singleSearchAfterAndAudit({
+        index: indexToUse,
+        operation: ReadOperations.Get,
+        aggs: {
+          active_alerts_bucket: {
+            date_histogram: {
+              field: ALERT_TIME_RANGE,
+              fixed_interval: fixedInterval,
+              hard_bounds: {
+                min: gte,
+                max: lte,
+              },
+              extended_bounds: {
+                min: gte,
+                max: lte,
+                min_doc_count: 0,
```

Comment:
I believe `min_doc_count` is a `date_histogram` property, the docs are not very clear on this haha 

```
{
  ...
  extended_bounds: { ... },
  min_doc_count: 0,
  ...
}
```